python/tchannel/io.py

```diff
@@ -19,8 +19,169 @@
 # THE SOFTWARE.
 
 from __future__ import absolute_import
+import os
+from .exceptions import StreamingException
+from .messages.common import StreamState
+
 
 try:
     from cStringIO import StringIO as BytesIO
 except ImportError:  # pragma: no cover
     from io import BytesIO  # noqa
+
+import tornado
+import tornado.gen
+import tornado.ioloop
+from tornado.iostream import PipeIOStream, StreamClosedError
+
+try:
+    from tornado.locks import Condition
+except ImportError:  # pragma: no cover
+    from toro import Condition
+
+from .messages import common
+
+
+class BaseArgStream(object):
+
+    def __init__(self):
+        """Base Arg Stream Class
+
+        _state is used to track the state of the arg stream.
```

Comment:
Don't need to mention this. It's obvious from the name. Also, `_state` is not part of the "public" interface of this class so it shouldn't be something exposed in the docstring.